src/java/test/org/apache/zookeeper/test/SessionTrackerCheckTest.java

```diff
@@ -185,8 +182,8 @@ public void testLeaderSessionTracker() throws Exception {
                 TICK_TIME, expirer.sid, false, testZKSListener());
 
         // Global session
-        sessionId = 0xdeadbeef;
-        tracker.addSession(sessionId, CONNECTION_TIMEOUT);
+        sessionId = 0xdeadbef0;
```

Comment:
Why have you changed this? :)

Reply:
I think this is not changed by intention, might happen when merge patch, I'll get this fixed.